formula/src/main/java/com/instacart/formula/Evaluation.kt

```diff
@@ -6,9 +6,20 @@ package com.instacart.formula
  * @param Output Usually a data class returned by formula that contains data and callbacks.
  * When it is used to render UI, we call it a render model (Ex: ItemRenderModel).
  *
- * @param updates A list of asynchronous events that formula wants to listen and respond to.
+ * @param actions A list of actions that will be performed by formula runtime.
  */
 data class Evaluation<out Output>(
     val output: Output,
-    val updates: List<Update<*>> = emptyList()
-)
+    val actions: List<BoundAction<*>> = emptyList()
```

Comment:
To have better naming downstream, it might be worth wrapping this into a data class?
```kotlin
data class Actions(val list: List<ICBoundAction<*>>)
```